### PR TITLE
Update features

### PR DESCRIPTION
**Pull request by arsalmalik1**

### CODE REVIEW

File: app/models/party.rb [hunk 1]
```diff
@@ -12,4 +12,7 @@ def self.is_vender(type)
   def self.get_party party_type
     is_vender(party_type) ? Party.vender : Party.dealer
   end
+  def current_date
```
Comment (mtmubeentahir) [inline app/models/party.rb:15]: Where we are using this method?

File: db/migrate/20201005135946_add_card_and_address_and_image_to_labor.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AddCardAndAddressAndImageToLabor < ActiveRecord::Migration[6.0]
+  def change
+    add_column :labors, :card, :string
```
Comment (mtmubeentahir) [inline db/migrate/20201005135946_add_card_and_address_and_image_to_labor.rb:3]: labors spelling mistake

File: app/views/stocks/edit.html.haml [hunk 3]
```diff
@@ -1,6 +1,9 @@
 - if @party_t == "dealer" || @party_t == "vender"
 	= render 'partials/top_btns', the_path: stock_path(product_type: @p_t)
-	%h1= "Editing " + @p_t + " Stock"
+	- if @p_t == nil
+		%h1= "Editing Stock"
+	- else
+		%h1= "Editing " + @p_t + " Stock"
```
Comment (mtmubeentahir) [inline app/views/stocks/edit.html.haml:6]: it's fine but it should be in a helper class.

File: db/migrate/20201005135946_add_card_and_address_and_image_to_labor.rb (hunk 2 above)
Comment (mutafaf) [reply db/migrate/20201005135946_add_card_and_address_and_image_to_labor.rb:3]: fixed already, not merged yet

**Pull request merged**